src/renderer/views/Watch/Watch.js

```diff
@@ -15,11 +15,11 @@ import FtAgeRestricted from '../../components/FtAgeRestricted/FtAgeRestricted.vu
 import packageDetails from '../../../../package.json'
 import {
   buildVTTFileLocally,
-  copyToClipboard,
+  copyToClipboard, deepCopy,
   extractNumberFromString,
   formatDurationAsTimestamp,
   formatNumber,
-  showToast
+  showToast, toDistractionFreeTitle
```

Comment:
Please place these on their own lines, I'm surprised the linter didn't complain about this.